mwdb/model/user.py

```diff
@@ -202,18 +202,22 @@ def set_group_admin(self, group_id, set_admin):
         member.group_admin = set_admin
 
     def has_access_to_object(self, object_id):
+        """
+        Query filter for objects visible by this user
+        """
         return object_id.in_(
             db.session.query(ObjectPermission.object_id).filter(
                 self.is_member(ObjectPermission.group_id)
             )
         )
 
-    def has_uploaded_object(self, object_id):
-        return object_id.in_(
-            db.session.query(ObjectPermission.object_id).filter(
-                and_(
-                    ObjectPermission.related_object == ObjectPermission.object_id,
-                    ObjectPermission.related_user_id == self.id,
-                )
-            )
+    def workspaces(self):
+        """
+        Query for workspace groups for this user
```

Comment:
```suggestion
        Get workspace groups for this user
```

Reply:
Not really because it doesn't return the workspace groups (just a query, there is not `all()` at the end)

Reply:
Oh, I see